src/wp-admin/includes/update-core.php

```diff
@@ -656,108 +655,60 @@ function update_core( $from, $to ) {
 }
 
 /**
- * Gets the checksums for the given version of ClassicPress.
+ * Preloads old Requests classes and interfaces.
  *
- * This function is a duplicate copy of `get_core_checksums()` to ensure the
- * new code is loaded and used when updating from a pre-1.3.0 version.
+ * This function preloads the old Requests code into memory before the
+ * upgrade process deletes the files. Why? Requests code is loaded into
+ * memory via an autoloader, meaning when a class or interface is needed
+ * If a request is in process, Requests could attempt to access code. If
+ * the file is not there, a fatal error could occur. If the file was
+ * replaced, the new code is not compatible with the old, resulting in
+ * a fatal error. Preloading ensures the code is in memory before the
+ * code is updated.
  *
- * @since CP-1.3.0
+ * @since 6.2.0
  *
- * @param string $version Version string to query.
- * @return bool|array False on failure. An array of checksums on success.
+ * @global array              $_old_requests_files Requests files to be preloaded.
+ * @global WP_Filesystem_Base $wp_filesystem       WordPress filesystem subclass.
+ * @global string             $wp_version          The WordPress version string.
+ *
+ * @param string $to Path to old WordPress installation.
  */
-function cp_get_core_checksums( $version ) {
-	$url = 'https://api-v1.classicpress.net/checksums/md5/' . $version . '.json';
-
-	$options = array(
-		'timeout' => wp_doing_cron() ? 30 : 3,
-	);
-
-	$response = wp_remote_get( $url, $options );
-
-	if ( is_wp_error( $response ) ) {
-		trigger_error(
-			sprintf(
-				/* translators: %s: support forums URL */
-				__( 'An unexpected error occurred. Something may be wrong with ClassicPress.net or this server&#8217;s configuration. If you continue to have problems, please try the <a href="%s">support forums</a>.' ),
-				__( 'https://forums.classicpress.net/c/support' )
-			) . ' ' . __( '(ClassicPress could not establish a secure connection to ClassicPress.net. Please contact your server administrator.)' ),
-			headers_sent() || WP_DEBUG ? E_USER_WARNING : E_USER_NOTICE
-		);
+function _preload_old_requests_classes_and_interfaces( $to ) {
```

Comment:
I updated both files in my instance and it did fix cosmetic issues on the update core page. However, when I run update, I get `Uncaught Error: Call to undefined function _preload_old_requests_classes_and_interfaces()`. Am I missing something?